CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 # CHANGELOG
 
-## v1.0.0-alpha.1
+## Unreleased
 
 *December 4, 2023*
```

Comment:
Do we need to update this date?

Reply:
We can deal with this when cutting the final `v1.0.0`